examples/example-gateway/app.go

```diff
@@ -62,14 +62,9 @@ func getRequestTags(ctx context.Context) map[string]string {
 func getRequestFields(ctx context.Context) []zap.Field {
 	var fields []zap.Field
 	headers := zanzibar.GetEndpointRequestHeadersFromCtx(ctx)
-	uuid, ok := headers["X-Uuid"]
```

Comment:
We should not change this section, we deliberately avoided to have token as log fields here.